_posts/2025-04-10-Super-Fast-Aggregations-in-OpenSearch-with-Star-tree-index.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title:  "The power of star-tree indexes: Supercharging OpenSearch aggregations" 
+authors:
+  - gbh
```

Comment:
These are github aliases, right?

Please correct the aliases:

  -  bharath-techie
  - sandeshkr419
  - backslasht
  - kolchfa

Reply:
@sandeshkr419 These are aliases to the author files https://github.com/opensearch-project/project-website/blob/main/_community_members/gbh.md